src/SFML/Window/macOS/HIDInputManager.hpp

```diff
@@ -286,7 +286,8 @@ class HIDInputManager
     ////////////////////////////////////////////////////////////
     // Member data
     ////////////////////////////////////////////////////////////
-    IOHIDManagerRef                                                       m_manager{}; ///< Underlying HID Manager
+    IOHIDManagerRef m_manager{};           ///< Underlying HID Manager
+    bool            m_keysInitializeTried; // Has initializeKeyboard been called at least once?
```

Comment:
```suggestion
    bool            m_hasInitializedKeys{}; // Has initializeKeyboard been called at least once?
```
Let's use a variable name that reads a bit more like English.